CMakeLists.txt

```diff
@@ -210,6 +210,17 @@ if(ENABLE_SHARED)
     )
     # Set the SOVERSION property of the wiredtiger library, so we can export the appropriately versioned symlinks.
     set_target_properties(wiredtiger_shared PROPERTIES SOVERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
+
+    # For MacOS builds we need to generate a dSYM bundle that contains the debug symbols for each 
+    # the WiredTiger library.
+    if (WT_DARWIN AND HAVE_DIAGNOSTIC)
```

Comment:
I think just `HAVE_DARWIN` would be better here. We could (in theory) have debug symbols without diagnostics.

Reply:
Yeah fair enough, but do you mean just `WT_DARWIN` or is `HAVE_DARWIN` a thing?

Reply:
Yeah I sneakily fixed it. You caught me anyway, though.